projects/elements-demo/src/app/app.module.ts

```diff
@@ -13,6 +13,7 @@ import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { ServiceWorkerModule } from '@angular/service-worker';
 import { environment } from '../environments/environment';
+import { GlobalErrorComponent } from './shared/global-error/global-error.component';
```

Comment:
Do we even need this ? Can;t we just re-use ` './shared/error/error.component';`

Reply:
I wanted to use two different ones, so that we can actually see it in action in the demo

Reply:
Ok makes sense ! 👍